coriolis/osmorphing/windows.py

```diff
@@ -68,6 +69,39 @@
 REQUIRED_DETECTED_WINDOWS_OS_FIELDS = [
     "version_number", "edition_id", "installation_type", "product_name"]
 
+INTERFACES_PATH_FORMAT = (
+    "HKLM:\\%s\\ControlSet001\\Services\\Tcpip\\Parameters\\Interfaces")
+
+STATIC_IP_SCRIPT_TEMPLATE = """
```

Comment:
In case Cloudbase-init doesn't do it already, please add `$ErrorActionPreference = "Stop"` at the beginning of the script.